src/Platform/OpenGL/Shaders/gbuffer_fragment.glsl

```diff
@@ -68,4 +70,5 @@ void main()
     OutAlbedo = vec4(fsin.RenderColor * albedo, emmisive / (emmisive + 1.0f));
     OutNormal = vec4(0.5f * normal + 0.5f, 1.0f);
     OutMaterial = vec4(parallaxOcclusion * occlusion, roughness, metallic, 1.0f);
+    OutSSRMask = vec4(float(receiveSSR),vec3(1.0));
```

Comment:
you can directly pass float to the shader instead of bool
with bool you have additional instruction in vertex shader:
```
%16 = OpSelect %float %13 %float_1 %float_0
%17 = OpCompositeConstruct %v4float %16 %float_1 %float_1 %float_1
```